model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -181,21 +166,20 @@ find_ie_bindings() {
 }
 
 if [[ $V_ENV -eq 1 ]]; then
-    "$python_binary" -m venv "$SCRIPTDIR/../venv${postfix}"
-    source "$SCRIPTDIR/../venv${postfix}/bin/activate"
-    venv_python_binary="$SCRIPTDIR/../venv${postfix}/bin/$python_binary"
+    "$python_binary" -m venv "$HOME/venv_mo${postfix}"
+    source "$HOME/venv_mo${postfix}/bin/activate"
+    venv_python_binary="$HOME/venv_mo${postfix}/bin/$python_binary"
+    # latest pip is needed to install tensorflow
+    "$venv_python_binary" -m pip install --upgrade pip
     $venv_python_binary -m pip install -r "$SCRIPTDIR/../requirements${postfix}.txt"
```

Comment:
May be wrap `$venv_python_binary` with double quotes? If the path contain spaces then the command will fail